automatic/selenium-chromium-edge-driver/update.ps1

```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  # We use the available package on CCR as the source of truth
-  $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
```

Comment:
As you're removing the call to this function, please remove the function itself.

Reply:
good point. I removed the function